File: src/main/java/com/cfgglobal/test/service/rule/access/AbstractAccessRule.java
package com.cfgglobal.test.service.rule.access;

import com.cfgglobal.test.service.rule.SecurityFilter;
import org.springframework.beans.factory.annotation.Autowired;


public abstract class AbstractAccessRule implements AccessRule {
    @Autowired
    protected SecurityFilter securityFilter;
}

File: src/main/kotlin/com/cfgglobal/test/service/rule/access/AbstractAccessRule.kt
package com.cfgglobal.test.service.rule.access

import com.cfgglobal.test.service.rule.SecurityFilter
import org.springframework.beans.factory.annotation.Autowired


abstract class AbstractAccessRule : AccessRule {
    @Autowired
    protected var securityFilter: SecurityFilter? = null
}
